documentation.html

```diff
@@ -0,0 +1,55 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Get Started | gotham UI</title>
+    <link rel="stylesheet" href="/style.css" />
+  </head>
+  <body>
+    <nav id="nav-bar">
+      <div id="nav-brand">
+        <h1>gotham<span>UI</span></h1>
+      </div>
+      <div id="nav-menu">
+        <div id="nav-home">home</div>
+        <div id="nav-docs">documentation</div>
+      </div>
+    </nav>
+    <div class="container">
+      <aside id="sidebar">
+        <section id="sidebar-get-started" class="sidebar-items">get started</section>
+        <section id="sidebar-avatar" class="sidebar-items">avatar</section>
+        <section id="sidebar-alert" class="sidebar-items">alert</section>
+        <section id="sidebar-badge" class="sidebar-items">badge</section>
+        <section id="sidebar-button" class="sidebar-items">button</section>
+        <section id="sidebar-card" class="sidebar-items">card</section>
+        <section id="sidebar-image" class="sidebar-items">image</section>
+        <section id="sidebar-input" class="sidebar-items">input</section>
+        <section id="sidebar-list" class="sidebar-items">list</section>
+        <section id="sidebar-navigation" class="sidebar-items">
+          navigation
+        </section>
+        <section id="sidebar-modal" class="sidebar-items">modal</section>
+      </aside>
+
+      <div class="inner-container">
+        <h2 class="inner-container-title">Get Started</h2>
+
+        <p class="inner-container-desc">
+          Lorem ipsum dolor sit amet consectetur, adipisicing elit. Error ad
+          harum voluptatem corrupti quis minima delectus tenetur facilis
+          reprehenderit rerum?
+        </p>
+        <div class="inner-container-type"> will update code soon </div>
+        <div class="inner-container-code">
+          <iframe
+            src="https://www.thiscodeworks.com/embed/620e1eb41285b600155ec22b"
+            class="code-snippet"
```

Comment:
Use this class name after iframe , eg. iframe class="code-snippet" src =""

Reply:
this issue  is in every component , will do all these minor changes soon